keyboards/exclusive/e7v1se/info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "e7v1se",
+    "url": "",
+    "maintainer": "Bart Riemens",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"k00", "x":0, "y":0},
+                {"label":"k01", "x":1, "y":0},
+                {"label":"k02", "x":2, "y":0},
+                {"label":"k10", "x":0, "y":1, "w":1.5},
+                {"label":"k12", "x":1.5, "y":1, "w":1.5}
```

Comment:
Please fill this out - the usual workflow (not sure if Merlin does this on his streams) is to recreate the layout in KLE, then use https://qmk.fm/converter to turn the raw JSON into an info.json.

Reply:
Done. Haven't been able to test tbh